packages/govern-subgraph/schema.graphql

```diff
@@ -24,15 +24,15 @@ type Govern @entity {
 type Execution @entity {
   id: ID!
   queue: GovernQueue!
-  payload: Payload!
+  container: Container!
   results: [Bytes!]
   createdAt: BigInt!
 }
 
 type GovernQueue @entity {
   id: ID!
   address: Bytes!
-  config: Config
+  configs: [Config!]
```

Comment:
I'd prefer to only keep a single config here—We could totally choose to track the history later, but let's keep it close to the contracts.

Reply:
I choose to keep track the history of configs because we need to know the specific configuration that a `Container` was schedule with. Otherwise when we update the config of the `Queue` the `Container` will have a the wrong configuration data.